fcrepo-http-api/src/main/resources/views/common.js

```diff
@@ -94,7 +94,15 @@
       const url = this.href;
       http('HEAD', url, function(res) {
         if (res.status >= 400 || res.getResponseHeader('Link') == null) {
-          location.href = url;
+          var newLocation = url;
```

Comment:
I am ok with settling on the current approach (with inclusion of warning comments) unless anyone feels strongly otherwise.
@escowles ? @acoburn ? @birkland ?